docs/contributor/04-30-ci-cd.md

```diff
@@ -32,19 +38,18 @@ The tests use an OAuth2 authorization server mock that is deployed internally in
-| [`PR Integration / Upgrade tests (pull_request)`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration.yaml#L65)          | Executes the upgrade integration test suite for the API Gateway module and verifies if the existing release can be successfully upgraded to the new version. |
-| [`Pull Request / Golang lint check (pull_request)`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-request.yaml#L26)            | Performs linting.                                                                                                                                            |
-| [`UI Tests / Wait for image build`]()                                                                                                                                                             | Waits for the operator image build to finish so that the image-dependent jobs can be executed.                                                               |
-| [`UI Tests / Run UI tests`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/ui-tests.yaml#L23)                                        | Runs UI tests for Busola extensions of the API Gateway module and APIRules.                                                                                  |
-| [`Pull Request / Run unit tests (pull_request)`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-request.yaml#L44)               | Runs unit tests with code coverage information.                                                                                                              |
-| [`Workflow validation / Ensure SHA pinned actions`](https://github.com/kyma-project/api-gateway/blob/main/.github/workflows/verify-commit-pins.yaml)                                              | Verifies that all external actions are pinned to a commit.                                                                                                   |
-| [`pull-gitleaks / scan`](https://github.com/kyma-project/api-gateway/blob/main/.github/workflows/pull-gitleaks.yaml#L10)                                                                          | Runs the Gitleaks scan to detect any leaks of secrets.                                                                                                       |
+| Name                                                                                                                                                             | Description                                                                                                                                                  |
```

Comment:
I remember that some time ago we decided to use commit hashes in these links so they don't break that often. Don't we want to continue using this approach? Other than that, lgtm